apps/client/src/app/pages/home/header/Header.tsx

```diff
@@ -0,0 +1,47 @@
+import NotificationIcon from '@home-assets/notifications-icon.png';
+import UserIcon from '@home-assets/user-icon.png';
+import Button from '@map-shared-components/Button';
+
+function Header({ isLoggedIn, username }: HeaderProps) {
+  const handleClick = () => {};
+
+  return (
+    <div className='flex items-center justify-between bg-[#1D3C66] text-[#CADEED] py-4 px-6 font-roboto'>
+      <div className='flex items-center sm:space-x-4'>
+        <h1 className='text-xl font-bold'>Hermes</h1>
+        <div className='space-x-4 hidden sm:block'>
```

Comment:
When we talk about responsive design, we do not consider to rest functionalities of the app, so maybe is a better idea to create a drop menu for Home About and Functions than only hide them